### DISCUSSION TITLE
Manually running frontend shows landing page

### DISCUSSION THREAD

**Discussion by zhangmain666** [RamiAwar/dataline#265] [Q&A] [answered]
I ran a manual build backend and frontend but the front end looked like a separate application, with no connection to the backend I started locally

Only the project introduction is displayed on the page, and the dialog Settings screen is not displayed
![9f360c9896c93ebfc334d389261e32d](https://github.com/user-attachments/assets/9e084433-8f87-4cfc-9386-5a6ed9a06ca5)

**Comment by RamiAwar** [comment, marked as answer]
Hey @zhangmain666 !

You just need to run the frontend with `NODE_ENV=local`

https://github.com/RamiAwar/dataline/blob/main/frontend%2FREADME.md

What you see is the website. We plan on separating those later.

Thanks for the feedback, we'll bring more attention to this in the docs!
